port/linux/omrvmem.c

```diff
@@ -192,7 +193,9 @@ initializeNumaGlobals(struct OMRPortLibrary *portLibrary)
 	PPG_numa_policy_mode = -1;
 	memset(&PPG_numa_mempolicy_node_mask, 0, sizeof(J9PortNodeMask));
 	mempolicyReturnCode = do_get_mempolicy(&PPG_numa_policy_mode, PPG_numa_mempolicy_node_mask.mask, maxNodes, (unsigned long)NULL, 0);
-
+	if ((0 != mempolicyReturnCode) && (EPERM == errno)) {
+		PPG_syscallNotAllowed = TRUE;
```

Comment:
Shouldn't this global have `numa` somewhere in the name or does this mean that no `syscall` can be made at all?

Reply:
@charliegracie I would like to rename it to something which states [these calls](https://docs.docker.com/engine/security/seccomp/#significant-syscalls-blocked-by-the-default-profile) are not allowed. NUMA is one of the consumer of these calls and someother functionality might also get benefit from this global if it needs to perform any of these calls.

I would like to take your view of how we could name it ? If we can just keep it to the NUMA context as currently its the only consumer of it we can add NUMA in it. Thank you

Reply:
@charliegracie I'm sorry i'm looking in the docker container perspective only, As OMR is not bound to any such things naming it having `numa` in it would be good (As we are using it in NUMA perspective only). Will `PPG_numaSyscallsNotAllowed` be fine ?